src/thyra/ThyraProvider.ts

```diff
@@ -185,6 +218,26 @@ export class ThyraProvider implements IProvider {
    * @returns a Promise that resolves to the details of the newly generated account.
    */
   public async generateNewAccount(name: string): Promise<IAccountDetails> {
```

Comment:
Considering your remark about the difference between the standard and what the wallet plugin is exposing, I guess that you should keep this function as unimplemented.